### PR TITLE
Lab Representative Page Push

### PR DESCRIPTION
**Pull request by franceme**
Added the page for lab representatives.

### CODE REVIEW

File: csgc-jekyll-src/_config.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ main_pages:
     title: "Officers"
   - url: "/Housing/"
     title: "Housing"
+  - url: "/labrepresentative/"
+    title: "Lab Representative"
```
Comment (ayaankazerouni) [inline csgc-jekyll-src/_config.yml]:
```suggestion
    title: "Lab Representatives"
```
You might also change the url for the sake of consistency.

File: csgc-jekyll-src/labrepresentative.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Lab Representative
+permalink: /labrepresentative/
+---
+
+## What's a Lab Representative do?
```
Comment (ayaankazerouni) [inline csgc-jekyll-src/labrepresentative.md]:
```suggestion
## What does a Lab Representative do?
```

**Comment by ayaankazerouni** [comment]
In the list of representatives, I don't think there's any need to include advisors who are unrepresented. Especially since the prose at the top says "if you don't see your advisor in this list..."

**Comment by franceme** [comment]
All of the changes have been and pushed.
The "empty" advisors were added since I auto converted the table.

**Comment by ayaankazerouni** [comment]
Yeah, I figured that's what happened for the empty advisors.

Thanks Myles!

**Pull request merged**